input/tunable_parameters/coriolis_test_parameters.in

```diff
@@ -0,0 +1,110 @@
+! $Id$
```

Comment:
Why do we need this file?  Can we just add any new parameters to 

https://github.com/larson-group/clubb_release/blob/master/input/tunable_parameters/tunable_parameters.in ?

Reply:
coriolis_test_parameters.in shuts off all the pressure terms so that Hing can compare with his analytic solution.